examples/helloworld-tutorial.md

```diff
@@ -129,7 +129,7 @@ At the root of your crate, create a `build.rs` file and add the following code:
 
 ```rust
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    tonic_build::prost::compile_protos("proto/helloworld.proto")?;
+    tonic_build::compile_protos("proto/helloworld.proto")?;
```

Comment:
these need to actually be reverted because of changes on master. Its fine for a bit to be out of sync with the latest release.